go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/submariner-io/lighthouse v0.16.7
 	github.com/submariner-io/shipyard v0.16.7
 	github.com/submariner-io/submariner v0.16.7
-	github.com/submariner-io/submariner-operator v0.16.7
+	github.com/submariner-io/submariner-operator v0.16.8-0.20240701075802-8fa04707dcc6
```

Comment:
This will need to be updated when https://github.com/submariner-io/submariner-operator/pull/3144 is merged.